cmake/finders/FindFFmpeg.cmake

```diff
@@ -93,10 +93,8 @@ The following cache variables may also be set:
 
 #]=======================================================================]
 
-# cmake-format: off
 # cmake-lint: disable=C0103
```

Comment:
Also remove all `cmake-lint` entries - that tool was part of `cmake-format` and `gersemi` takes care of some amount of linting itself.

Reply:
Removed.

Reply:
Were those the only remaining occurrences of `cmake-lint`?

Reply:
Yes, according to git grep at least.